corehq/apps/consumption/shortcuts.py

```diff
@@ -97,12 +97,12 @@ def build_consumption_dict(domain):
     SQLDefaultConsumption.objects.filter(domain=domain)
 
     return dict(
-        (tuple(
+        (tuple([
```

Comment:
nit: could use a literal tuple
```suggestion
        ((
```